website/docs/r/vapp_firewall_rules.html.markdown

```diff
@@ -0,0 +1,119 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vapp_firewall_rules"
+sidebar_current: "docs-vcd-resource-vapp-firewall-rules"
+description: |-
+  Provides a vCloud Director vApp Firewall resource. This can be used to create, modify, and delete firewall settings and rules.
+---
+
+# vcd\_vapp\_firewall\_rules
+
+Provides a vCloud Director vApp Firewall resource. This can be used to create,
+modify, and delete firewall settings and rules.
+
+~> **Note:** Using this resource overrides any existing firewall rules on vApp network.
```

Comment:
Useful note! What about making it a warning, so it stands out more?

Reply:
changed